server/memo.go

```diff
@@ -53,6 +53,25 @@ func (s *Server) registerMemoRoutes(g *echo.Group) {
 			}
 		}
 
+		// Find system settings
```

Comment:
@itschip Please check out what I have changed in the commit.

https://github.com/usememos/memos/pull/1003/commits/0de531ecf2481f25ec8bca46f1c4def8eb31ac97

Reply:
I was a bit confused what to do here! Thanks :)